src/GafferUI/StandardNodeGadget.cpp

```diff
@@ -763,14 +764,27 @@ float StandardNodeGadget::focusBorderWidth() const
 	return min( g_maxFocusWidth, max( 0.75f, 8.0f * pixelSize ) );
 }
 
+void StandardNodeGadget::setHighlighted( bool highlighted )
+{
+	NodeGadget::setHighlighted( highlighted );
+	m_highlightedStandardNodeGadget = highlighted;  // Don't really get why the base class one is protected
```

Comment:
Why do we need to duplicate the state? I think we can just call `getHighlighted()` when we want to access it can't we?